rules/windows/registry_event/sysmon_runonce_persistence.yml

```diff
@@ -0,0 +1,25 @@
+title: Run Once task configuration in registry - Persistence method
+id: c74d7efc-8826-45d9-b8bb-f04fac9e4eff
+description: Rule to detect the configuration of Run Once registry key. Configured payload can be run by runonce.exe /AlternateShellStartup
+author: Avneet Singh @v3t0_
```

Comment:
Hey @V3T0, thanks for your nice contribution!

Handles with the '@' char must be put in quotes

Reply:
Thanks for your comments. I'll put the twitter handle in the quotes